client/styles/components/_console.scss

```diff
@@ -102,4 +127,15 @@
 	.preview-console--collapsed & {
 		display: none;
 	}
+}
+
+.preview-console__logged-times {
+	font-size: 10px;
```

Comment:
This project doesn't use pixel measurements but uses REM's! Therefore this should be
```SCSS
  font-size: #{10 / $base-font-size}rem;
```